### DISCUSSION TITLE
How to have dynamic segments without showing them in URL

### DISCUSSION THREAD

**Discussion by laurent512** [vercel/next.js#43847] [Help]
Hi, 

Next JS version  : 13 (if it matters)

I am building a dashboard allowing users to view data about their own real estate properties. 
For example, the user can click on left side bar navigation menu to see electricity cost in the property area. 
 
The twist is that the dasboard structure should always remain the same and the users should be able to switch between different properties (from top horizontal header menu). 

I could use dynamic segments like /property/[pid]/electricity , but I would prefer to keep /property/electricity . 
is there a elegant solution to do that ? Thank you so much

**Comment by ciruz** [comment]
Checkout ``Link`` with the `as` attribute or ``Router``, the `push` method also has an `as` attribute. The `as` attribute is basically the URL what the user will see in his browser.

Be aware this is not available in next 13 - `/app` directory.
